packages/react/src/components/AccountSettings/types.ts

```diff
@@ -13,11 +13,13 @@ import {
  *
  * Note that `PrimitiveProps` is used to get native html types, like `onSubmit`.
  */
-type CommonPasswordFieldProps = PrimitiveProps<PasswordFieldProps, 'input'>;
-type CommonTextFieldProps = PrimitiveProps<TextFieldProps, 'input'>;
-type CommonImageProps = PrimitiveProps<ImageProps, 'img'>;
-type CommonButtonProps = PrimitiveProps<ButtonProps, 'button'>;
-type CommonAlertProps = PrimitiveProps<AlertProps, 'div'>;
+type CommonPasswordFieldProps = Partial<
+  PrimitiveProps<PasswordFieldProps, 'input'>
+>;
+type CommonTextFieldProps = Partial<PrimitiveProps<TextFieldProps, 'input'>>;
+type CommonImageProps = Partial<PrimitiveProps<ImageProps, 'img'>>;
+type CommonButtonProps = Partial<PrimitiveProps<ButtonProps, 'button'>>;
+type CommonAlertProps = Partial<PrimitiveProps<AlertProps, 'div'>>;
```

Comment:
Made these partial, so that override components don't require certain primitive props.

Reply:
I don't believe so, if they use primitives directly like so:

```ts
const components = {
  NewPassword: () => (<PasswordField />)
}
```

they will get an TS error because they don't have the required field `label`.

If they do 
```ts
const components = {
  NewPassword: (props) => (<PasswordField />)
}
```

or just 

```ts
const components = {
  NewPassword: PasswordField
}
```

all the required props will be spread so those would also work.

Reply:
If they do 
```ts
const components = {
  NewPassword: (props) => (<PasswordField />)
}
```

or just 

```ts
const components = {
  NewPassword: PasswordField
}
```

all the required props will be spread so those would also work.